fixity/utils.py

```diff
@@ -18,7 +19,7 @@ def check_valid_uuid(uuid):
     This uses the UUID() class constructor from the uuid module,
     which raises a ValueError if the passed string is not a valid UUID.
     """
-    if not isinstance(uuid, basestring):
+    if not isinstance(uuid, six.string_types):
```

Comment:
Since this is all we're using six for, maybe it would be better to avoid the dependency and do this manually?